server/server.js

```diff
@@ -15,17 +14,15 @@ app.use(compression());
 app.use(logger);
 
 if (process.env.NODE_ENV === "production") {
-  app.use(express.static("build", {
-    // etag: false
-  }));
+  app.use(express.static("build"));
 } else {
   app.use(devMiddleware);
   app.use(hotMiddleware);
 }
 
 app.get("*", (req, res) => {
   if (process.env.NODE_ENV === "production") {
-    res.write(fs.readFileSync(path.resolve("build/index.html")));
+    res.sendFile(path.join(__dirname + "/build/index.html"));
```

Comment:
Concatenating paths with `+` will break on Windows. You using `path.join` break already. `+` should be replaced with `,`.